ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageCveSingle.test.js

```diff
@@ -89,24 +90,22 @@ describe('Workload CVE Image CVE Single page', () => {
         // Wait for the loading spinner to disappear
         cy.get('.pf-c-spinner').should('not.exist');
 
-        cy.get(`${selectors.firstTableRow} td[data-label="Namespace"]`).then(([$namespace]) => {
-            const namespace = $namespace.innerText;
+        const namespace = 'stackrox';
 
-            selectResourceFilterType('Namespace');
-            typeAndEnterResourceFilterValue('Namespace', `bogus-${namespace}`);
+        selectResourceFilterType('Namespace');
+        typeAndCreateResourceFilterValue('Namespace', `bogus-${namespace}`);
 
-            cy.get(`table tbody tr td[data-label="Namespace"]:contains("${namespace}")`).should(
-                'not.exist'
-            );
+        cy.get(`table tbody tr td[data-label="Namespace"]:contains("${namespace}")`).should(
+            'not.exist'
+        );
 
-            cy.get(selectors.clearFiltersButton).click();
+        cy.get(selectors.clearFiltersButton).click();
 
-            typeAndEnterResourceFilterValue('Namespace', namespace);
+        typeAndSelectResourceFilterValue('Namespace', namespace);
 
-            cy.get(
-                `table tbody tr td[data-label="Namespace"]:not(:contains("${namespace}"))`
-            ).should('not.exist');
-        });
+        cy.get(`table tbody tr td[data-label="Namespace"]:not(:contains("${namespace}"))`).should(
+            'not.exist'
+        );
```

Comment:
This was changed from grabbing the namespace from the first table row, to using an explicit namespace of `stackrox`. The problem with the former is that a namespace name that is a prefix of another namespace name will cause test failures.

This was an issue locally and could potentially be an issue in CI. Using `stackrox` should lessen the chance that this happens without losing any test functionality.

Reply:
Super, you are thinking ahead.

For information, not a change request. Because of central versus central-db, and in case of something like stackrox and stackrox-operator in the future, some tests replace `contains` pseudo-selector with `contains` method:
* https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/vulnmanagement/VulnerabilityManagement.helpers.js#L312-L313
* https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/vulnmanagement/VulnerabilityManagement.helpers.js#L64-L77

As a bonus, it returns first found without need for `:first` pseudo-selector.

https://docs.cypress.io/api/commands/contains

Reply:
+1, good idea. I had thought about switching to the `contains` method but was aiming to keep the changes small for this CI-repair pull request.

I didn't know that `contains()` automatically selected the first element, so that is a nice bonus as well. I'll likely lean on this more than the pseudo element in the future tests and hopefully be able to refactor these a bit in the process.

Reply:
When traveling the road paved with good intentions, I broke this test in a different way. I pushed an updating reverting this change, but then using the `contains()` method you mentioned in order to do a regex match instead of a substring match.